test/k8sT/Nightly.go

```diff
@@ -357,6 +358,20 @@ var _ = Describe("NightlyExamples", func() {
 	})
 
 	Context("Cilium DaemonSet from example", func() {
+		// validatedImage validates that the cilium image is the same as the
```

Comment:
Make this more clear by specifying that we are checking against the Cilium image which is being used in currently running pods.